pkg/common/version/version.go

```diff
@@ -7,12 +7,23 @@ const (
 )
 
 var (
-	gittag = ""
+	gittag  = ""
+	githash = ""
 )
 
 func Version() string {
 	if gittag == "" {
-		return fmt.Sprintf("%s-dev", Base)
+		// Metrics systems sometimes automatically reject values with full SHA1 hashes
+		// since they appear to be high cardinality.
+		// For our purposes, the first 7 chars of the SHA should be enough.
+		var githashShort string
+		if githash == "" {
+			githashShort = "unk"
+		} else {
+			githashShort = githash[:7]
```

Comment:
Looks like the git command has a flag for this
```
git rev-parse --short=7 HEAD
```
One advantage of using the flag is that if more than 7 characters are needed for the reference to be unique, it will adjust as needed.